crates/types/src/services/executor.rs

```diff
@@ -294,6 +295,10 @@ pub enum Error {
     RelayerGivesIncorrectMessages,
     #[display(fmt = "Consensus parameters not found for version {_0}")]
     ConsensusParametersNotFound(ConsensusParametersVersion),
+    #[display(fmt = "Invalid WASM bytecode: {_0} (version: {_1:?})")]
+    InvalidWasm(String, Option<StateTransitionBytecodeVersion>),
+    #[display(fmt = "WASM support is not enabled")]
+    NoWasmSupport,
```

Comment:
We only can append new errors into this error

Reply:
Removed the variants in 71744b63c4e1f8ca2935baeb953653a85e2a0275, they now live in a separate enum.